docs/examples/plot_hamersley.py

```diff
@@ -30,8 +30,10 @@
     use_australian_state_data="WA",
     working_projection="EPSG:28350",
     bounding_box=bbox_3d,
+    clut_file_legacy=False,
```

Comment:
This is already the default so isn't required.

Reply:
yep, in https://github.com/Loop3D/map2loop/pull/60/commits/450b372cfc43063a2de2eebf2b6e05458114c425